### DISCUSSION TITLE
帮忙看一下一个文件

### DISCUSSION THREAD

**Discussion by wudiwudidream** [Meloong-Git/PCL#4516] [提问] [answered]
堆栈文件，由游戏导出来的一个文件
[s.txt](https://github.com/user-attachments/files/16636884/s.txt)
这个里面是有十进制的，还请有懂的帮忙解答一下

**Comment by hejiehao** [comment]
若你使用PCL2启动器，在游戏崩溃后在崩溃界面点左下角`导出错误报告`，然后把导出的压缩包整个发上来 ![Image](https://img.kookapp.cn/assets/2023-03/CqLof1HzCe0g405l.png)

**Reply by wudiwudidream** [reply]
> 若你使用PCL2启动器，在游戏崩溃后在崩溃界面点左下角`导出错误报告`，然后把导出的压缩包整个发上来 ![Image](https://camo.githubusercontent.com/74bde40aba49b0ae20bbbdcb047c2f1baf7c7821f325909c6dbc92498f83bcef/68747470733a2f2f696d672e6b6f6f6b6170702e636e2f6173736574732f323032332d30332f43714c6f6631487a436530673430356c2e706e67)

这个是堆栈报告（游戏导出来的）

**Comment by MoYuan-CN** [comment, marked as answer]
堆栈报告让我们怎么看…… 这东西只有开发者对着报告所表达的代码位置才能看懂……

**Comment by hejiehao** [comment]
可以加入 QQ 群 666546887 来寻求帮助
